문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,5 @@
+function solution(s) {
+    s = s.split("").sort().reverse().join("");
+
+    return s;
```

Comment:
```suggestion
    return s.split('').sort().reverse().join('');
```
s라는 변수를 return 외에 사용하지 않으니 리턴문에 저 로직을 써주는게 더 좋을거같아요.

Reply:
감사합니다 👍👍